### PR TITLE
refactor: reformat project

### PR DESCRIPTION
**Pull request by izayl**
**what this pr do?**

- reformatted project readme
- reordered project example folders
- add issue template

### CODE REVIEW

**Review by yingjingyang — COMMENTED** [review]
目录结构改动太大

File: README.md [hunk 1]
```diff
@@ -1,204 +1,132 @@
...
+- 以太坊区块链浏览器: <https://kovan.etherscan.io>
+- JSON-PRC接口: <https://eth.wiki/json-rpc/API>
+- tenderly合约验证 <https://dashboard.tenderly.co/explorer>
+- remix本地环境 <https://zhuanlan.zhihu.com/p/38309494>
+
+## 基础任务 - Basic Tasks
+
+通过以下基础任务，了解开发 Dapp 的基本工具和开发知识。
+
+- 01. [use web3.js deploy contract](courses/basic/01-web3js-deploy/README.md)
```
Comment (maroonstar) [inline README.md:47]:
```suggestion
01. [use web3.js deploy contract](courses/basic/01-web3js-deploy/README.md)
```

Comment (maroonstar) [reply README.md:47]: Use number directly

**Review by maroonstar — CHANGES_REQUESTED** [review]
Good job !
Please use Number serial number

**Review by maroonstar — APPROVED** [review]

**Review by yingjingyang — APPROVED** [review]

**Pull request merged**